### ISSUE TITLE
"Unhandled opcode: 190" and failure to purchase new tickets and transfer funds

### ISSUE THREAD

**Issue by hb9xar** [decred/dcrwallet#1057]
I have separate wallets for ticket purchase (1 wallet, non-redundant) and voting (2 wallets running in parallel on different servers). 
Since upgrading to the master from 2018-02-23 I get errors when my "purchasing wallet" tries to purchase a new ticket:

$ dcrctl purchaseticket "default" 85 1 "Ds..<my address of the voting wallet>" 1
-4: failed to send split transaction: -1: Rejected transaction 252b6670664831824e655ff3ebe0d7d029aeda261a4431b9d41bff1190fa42a3: Tx 252b6670664831824e655ff3ebe0d7d029aeda261a4431b9d41bff1190fa42a3 attempted to spend an OP_SSTX tagged output, however it was not an SSGen or SSRtx tx; SSGen err: SSGen tx did not include a stakebase in the zeroeth input position, SSRtx err: SSRtx has an  invalid number of inputs

Also I see frequent logs in the purchasing AND voting wallets, eg. when doing an getbalance operation:
"WLLT: Unhandled opcode: 190"

I have recreated the Wallet from seed, but the problem persists.
The same happens if I try to do an sendtoaddress to another DCR address.

With an *older* dcrwallet (dated 2017-11-30) and the same purchasing wallet (recreated from seed), ticket purchase and sendtoaddress operations do work as expected.

On an similar Testnet setup (also one purchasing wallet and two voting wallets) I don't see this issue.

**Comment by jrick** [comment]
Are you able to pinpoint which commit introduces the issue with `git bisect`?

**Comment by hb9xar** [comment]
So far I have never used git bisect - I may need some assistance with that.

There are still some DCR in the wallet in question, so transfers may be still be tested.
Aside from that, the mentioned "WLLT: Unhandled opcode: 190" do pop up on voting only wallets as well.

**Comment by jrick** [comment]
Ok, I don't know about the opcode warning (190 is OP_CHECKSIGALT which is used by schnorr and edwards pay-to-pubkey and p2pkh output scripts, have you ever used those?) but I see the cause for the other issue.  We made a db upgrade and after closer inspection, it wrote data to the wrong location and now it's picking unusable outputs to spend.  The error is recoverable but will require an additional database upgrade to resolve.  In the meantime, you can either run the wallet with any commit before 023f5bad0e51fec78df3eff9fb89add893b3fc32.

**Comment by jrick** [comment]
Can you test #1059?

**Comment by hb9xar** [comment]
Great, I just tested this pull request  (first on testnet and then on the affected mainnet wallet).
It fixes the issues I observed (including the Unhandled opcode warning).
